app/views/admin/editions/_edit_attachments.html.erb

```diff
@@ -16,7 +16,7 @@
           <%= f.text_field :title %>
         <% end %>
       </td>
-      <td class="attachment-filename"><%= link_to_attachment(attachment, preview: !@edition.published?, truncate: @edition.imported?) %></td>
+      <td class="attachment-filename"><%= link_to_attachment(attachment, preview: !@edition.published?, truncate: false) %></td>
```

Comment:
It looks like this was the only place that the `truncate:` option was being set to `true`.

So it'd be great to remove it from the link_to_attachment helper altogether ✨🧹

https://github.com/alphagov/whitehall/blob/78e4f19186d2e7ba1ec0cdea7a657a996280646f/app/helpers/application_helper.rb#L47-L54